### PR TITLE
Submit extension flow

### PR DESCRIPTION
**Pull request by quininez**
## Description
* publish extension flow
* submit release flow

## Checklist

- [x] I have read the **README** document
- [x] I have read the **CONTRIBUTING** document
- [ ] I have added tests to cover my changes
- [ ] All new and existing tests passed
- [x] My commit message follows [Conventional Commit Guideline](https://conventionalcommits.org/)

### CODE REVIEW

File: extensions/apps/extensions/src/components/pages/extension-submit-page.tsx [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+
+  const {
+    data: { authenticatedDID },
+  } = useAkashaStore();
+
+  const [createAppMutation, { loading }] = useCreateAppMutation({
+    context: { source: sdk.current.services.gql.contextSources.composeDB },
+    onCompleted: () => {
+      //remove published extension from list of local extensions
+      const newLocalDraftExtensions = draftExtensions.filter(
```
Comment (kenshyx) [inline extensions/apps/extensions/src/components/pages/extension-submit-page.tsx:63]: add some documentation about what is happening at this level

File: extensions/apps/extensions/src/components/pages/release-submit-page.tsx [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+
+  const handleClickSubmit = appReleaseFormData => {
+    const appReleaseContent = {
+      applicationID: extensionId,
+      version: appReleaseFormData?.versionNumber,
+      source: appReleaseFormData?.sourceURL,
+      createdAt: new Date().toISOString(),
+      meta: [
+        {
+          provider: 'AkashaApp',
```
Comment (kenshyx) [inline extensions/apps/extensions/src/components/pages/release-submit-page.tsx]: These values should be enums or constants that are exported.

File: extensions/apps/extensions/src/components/pages/extension-submit-page.tsx [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+      event: NotificationEvents.ShowNotification,
+      data: {
+        type: NotificationTypes.Info,
+        title,
+      },
+    });
+  }, []);
+
+  const draftExtensions: Extension[] = useMemo(
+    () => JSON.parse(localStorage.getItem(`${DRAFT_EXTENSIONS}-${authenticatedDID}`)) || [],
```
Comment (kenshyx) [inline extensions/apps/extensions/src/components/pages/extension-submit-page.tsx]: JSON.parse should be always contained in a try catch block

**Comment by SeverS** [comment]
Pressing enter while editing a field automatically submits the form...
Because of this, one of my apps broke with error:
```
index.js:1445 TypeError: Cannot read properties of null (reading 'src')
    at src_components_index…x.index.js:2:119384
    at Array.map (<anonymous>)
    at src_components_index…x.index.js:2:119307
    at Id (react-dom.production.min.js:165:137)
    at Xb (react-dom.production.min.js:200:284)
    at Rk (react-dom.production.min.js:198:114)
    at xb (react-dom.production.min.js:196:166)
    at Li (react-dom.production.min.js:189:373)
    at db (react-dom.production.min.js:79:182)
    at react-dom.production.min.js:184:289
```
source-code:
```
r.useEffect)(( () => {
                p?.gallery?.length > 0 && v(p?.gallery?.map((e => {
                    const t = (0,
                    l.transformSource)(e);
                    return {
                        ...e,
                        src: e.src,
                        displaySrc: t.src, // <------------ THIS IS THE CULPRIT -----
                        size: {
                            height: e.height,
                            width: e.width
                        }
                    }
                }
                )))
            }
```

**Review by SeverS — DISMISSED** [review]

**Review by kenshyx — APPROVED** [review]

**Review by SeverS — APPROVED** [review]

**Pull request merged**